### PR TITLE
ceph.spec.in: Fedora > 20 has junit instead of junit4

### PR DESCRIPTION
**Pull request by dachary**
http://tracker.ceph.com/issues/10728 Fixes: #10728

Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

**Comment by dachary** [comment]
@ktdreyer ping ?

**Comment by ktdreyer** [comment]
Apologies for the delay in reviewing.

I looked into this, and the package was renamed a long time ago (around the Fedora 15 timeframe). The "junit4" name is only relevant for EPEL 6. For EPEL 7 and Fedora 20, the "junit" package has "`Provides: junit4`". And most recently, in the junit package that ships in Fedora 21 and 22, the package maintainer dropped the old `Provides:` line in [this Fedora dist-git commit](http://pkgs.fedoraproject.org/cgit/junit.git/commit/?id=c9b62c4b0148a0e9bfd2d1375f1c5d57ee10fd4d).

In other words, you can change ceph.spec.in to only use "junit4" `%if 0%{?el6}`, and use "junit" otherwise.

**Comment by dachary** [comment]
Most impressive diagnostic, thanks, will do :-)

**Comment by dachary** [comment]
@ktdreyer applied, rebased and repushed

**Comment by ktdreyer** [comment]
Awesome, thanks @dachary !
